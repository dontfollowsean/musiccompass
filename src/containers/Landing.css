.video_container iframe {
	position: absolute;
	border: 0;
	top: 0;
	left: 0;
	width: 100%;
	height: 100%;
	z-index: -99;
}
