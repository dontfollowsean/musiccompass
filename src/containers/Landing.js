// @flow

import React from 'react';
import SearchBox from '../components/SearchBox';

const Landing = () => (
	<div className="landing">
		<h1>Travel with music</h1>
		<h3>Create the local playlist of your dreams. Try it! </h3>
		<SearchBox />
	</div>
);

export default Landing;
