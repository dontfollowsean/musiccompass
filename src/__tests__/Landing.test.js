import React from 'react';
import { shallow } from 'enzyme';
import Landing from '../containers/Landing';

test('renders without crashing', () => {
	const component = shallow(<Landing />);
	expect(component).toMatchSnapshot();
});
