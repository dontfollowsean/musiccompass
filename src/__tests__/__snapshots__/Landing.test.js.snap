// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<div
  className="landing"
>
  <h1>
    Travel with music
  </h1>
  <h3>
    Create the local playlist of your dreams. Try it! 
  </h3>
  <SearchBox />
</div>
`;
